### PR TITLE
try to fix the build

### PR DESCRIPTION
**Pull request by derhuerst**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: node_js
+before_install:
+  - npm i -g npm@4
```
Comment (derhuerst) [inline .travis.yml]: Note that we need to revert this change once `npm@5` is reasonably stable.

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
 	},
 	"scripts": {
 		"test": "node test.js | tap-min",
-		"prepublish": "npm test"
+		"prepublishOnly": "npm test"
```
Comment (derhuerst) [inline package.json]: Note that we can revert this change for `npm@>=6` (which doesn't exist yet).

**Comment by derhuerst** [comment]
Seems like the build now fails because of DB.

@juliuste Will put [`tap-spec`](https://github.com/scottcorgan/tap-spec#readme) if you don't mind, so we can see *what* fails.

cc @highsource 😉

**Comment by juliuste** [comment]
Is this outdated?

File: .travis.yml (hunk 1 above)
Comment (derhuerst) [inline .travis.yml]: `npm@5.2.0` is now reasonably stable as it seems.

**Comment by derhuerst** [comment]
> Is this outdated?

Don't think so. The `prepublish` -> `prepublishOnly` change will still be useful until `npm@6` or `npm@7`. (It lets Travis run the `build` script only *once*, not as part of `npm install`)

Will change to `npm@5` again.

**Comment by derhuerst** [comment]
bump.
